controller/embedded/api-schema/entities/ca.put.json

```diff
@@ -25,6 +25,10 @@
       "type": "boolean",
       "default": false
     },
+    "identityRoles": {
```

Comment:
@plorenz I believe this is a good attribute name but let me know if not.

Reply:
Makes sense to me

Reply:
`defaultIdentityRoles` ?